### PR TITLE
bug/322: Fixed unstable link order in dependencies.md

### LINKED ISSUE #322: Order of links in dependencies.md is unstable
In https://github.com/exasol/exasol-test-setup-abstraction-server/pull/17 the local order of the links was different then on the CI build.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #322

### CODE REVIEW

File: doc/changes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Changes
 
+* [2.5.0](changes_2.5.0.md)
```
Comment (kaklakariada) [inline doc/changes/changelog.md:3]: This is just a bugfix, an increment of the patch version should be enough. Or do you plan other features?

**Review by kaklakariada — APPROVED** [review]

File: doc/changes/changelog.md (hunk 1 above)
Comment (jakobbraun) [reply doc/changes/changelog.md:3]: yea, I would release it end of this week and think until then we will get a feature done.

File: .github/workflows/release_droid_prepare_original_checksum.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
             ${{ runner.os }}-go-${{ matrix.go }}
             ${{ runner.os }}-go-
       - name: Install Go tools
-        run: go install github.com/google/go-licenses@latest
+        run: go install github.com/google/go-licenses@v1.2.1
```
Comment (ckunki) [inline .github/workflows/release_droid_prepare_original_checksum.yml:35]: Would it make sense to define the actual version in a single place a use only references in the *.yml files to this version?

Comment (jakobbraun) [reply .github/workflows/release_droid_prepare_original_checksum.yml:35]: yes, we have an issue for installing it automaticlly.

Comment (jakobbraun) [reply .github/workflows/release_droid_prepare_original_checksum.yml:35]: https://github.com/exasol/project-keeper/issues/323

**Pull request merged**